test/inspector_controller_test.dart

```diff
@@ -551,7 +528,7 @@ void main() async {
       // TODO(jacobr): would be nice to have some tests that trigger a hot
       // reload that actually changes app state in a meaningful way.
 
-      await tearDownEnvironment();
+      await env.tearDownEnvironment(force: true);
```

Comment:
I think the force can be removed from this case?

Reply:
Done. added a forced tear down in tearDownAll